dlt/destinations/impl/filesystem/sql_client.py

```diff
@@ -168,12 +170,17 @@ def _escape_column_name(col_name: str) -> str:
             first_file_type = self.get_file_format(table_schema)
 
             # build files string
-            supports_wildcard_notation = not self.is_abfss
+            supports_wildcard_notation = not self.is_abfss and is_folder
```

Comment:
does this duckdb limitation apply to all protocols? did you check it? could be useful

Reply:
It does, and that is also reflected in the statement. supports_wildcard_notation always if false if we do not have a folder, regardless of the protocol.